src/test/java/herokuapp/InputsTests.java

```diff
@@ -0,0 +1,50 @@
+package herokuapp;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public class InputsTests {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+    }
+
+    @AfterClass
+    public void tearDown() {
+        driver.quit();
+    }
+
+    @Test
+    public void inputsTest() {
+        driver.get("http://the-internet.herokuapp.com/inputs");
+        driver.findElement(By.xpath("//input")).sendKeys(Keys.ARROW_UP);
+        String upKeyResult = driver.findElement(By.xpath("//input")).getAttribute("value");
+        assertEquals(upKeyResult, "1");
+        driver.findElement(By.xpath("//input")).sendKeys(Keys.ARROW_DOWN, Keys.ARROW_DOWN, Keys.ARROW_DOWN);
```

Comment:
You don't need to locate this element again and again until page is reloaded.
You can find it once at the very beginning and then reuse.